### PR TITLE
Added rockandroll

### LINKED ISSUE #1: Basic praise function
So I thought for starters we could have something like: 

```r
praise <- function(template = "This is ${superb}!") {
  ## ...
}
```

and then `praise` just inserts a random adjective from `superb` into the template. 

?

### PR DESCRIPTION
**Pull request by brezniczky**
Added rockandroll, closes #1.

### CODE REVIEW

File: R/adjective.R [hunk 1]
```diff
@@ -88,6 +88,7 @@ adjective <- c(
   "primo",
   "proper",
   "rad",
+  "rockandroll",
```
Comment (gaborcsardi) [inline R/adjective.R]: :) Thanks! Could you please move it after `riveting`, so that it is in alphabetical order.

Comment (brezniczky) [inline R/adjective.R]: Sure, have hopefully fixed it - however, the CI system is complaining, maybe for reasons external to this change.

**Comment by gaborcsardi** [comment]
Thanks!

**Comment by gaborcsardi** [comment]
The tests fail because I didn't expect smileys, I'll fix them soon.

**Pull request merged**